apps/src/lib/node/ledger/shell/finalize_block.rs

```diff
@@ -830,22 +830,23 @@ where
         pgf_fundings.sort_by(|a, b| a.id.cmp(&b.id));
 
         for funding in pgf_fundings {
-            if credit_tokens(
+            if storage_api::token::transfer(
```

Comment:
I don't think this is complete, we also need to update/change https://github.com/anoma/namada/blob/main/apps/src/lib/node/ledger/shell/finalize_block.rs#L858 - I think we can actually use `credit_tokens` there, but then we shouldn't send the steward inflation to the PGF account in the first place (https://github.com/anoma/namada/blob/main/apps/src/lib/node/ledger/shell/finalize_block.rs#L812)